### PR TITLE
feat(dynamic-sampling) add logging to specific task for debugging

### PR DESCRIPTION
**Pull request by shellmayr**

### CODE REVIEW

**Review by antonpirker — CHANGES_REQUESTED** [review]
please use the top level api, makes it simpler :-)

**Review by iambriccardo — APPROVED** [review]
LGTM

File: src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py [hunk 1]
```diff
@@ -88,6 +90,10 @@
 )
 @dynamic_sampling_task_with_context(max_task_execution=MAX_TASK_SECONDS)
 def boost_low_volume_projects(context: TaskContext) -> None:
+    current_scope = sentry_sdk.Scope.get_current_scope()
+    traceparent = current_scope.get_traceparent()
```
[the remark this answers is not in the document]
Comment (shellmayr) [reply src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py]: Changed according to suggestion

File: src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py [hunk 2]
```diff
@@ -88,6 +90,10 @@
...
+    baggage = current_scope.get_baggage()
```
[the remark this answers is not in the document]
Comment (shellmayr) [reply src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py]: Changed according to suggestion

File: src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py [hunk 3]
```diff
@@ -114,6 +120,10 @@ def boost_low_volume_projects_of_org(
         tuple[ProjectId, int, DecisionKeepCount, DecisionDropCount]
     ],
 ) -> None:
+    current_scope = sentry_sdk.Scope.get_current_scope()
+    traceparent = current_scope.get_traceparent()
+    baggage = current_scope.get_baggage()
```
[the remark this answers is not in the document]
Comment (shellmayr) [reply src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py]: Changed according to suggestion

**Review by antonpirker — APPROVED** [review]
looking good!

**Pull request merged**